core/src/cluster_nodes.rs

```diff
@@ -462,22 +479,62 @@ impl From<Pubkey> for NodeId {
     }
 }
 
+// pub for bench
+pub fn make_cluster<R: Rng>(
```

Comment:
maybe rename this to `make_rand_cluster` to indicate this is a test/simulation only function.

Reply:
changed to `make_test_cluster`